pytest_plugins/fixture_markers.py

```diff
@@ -6,7 +6,10 @@
 
 
 def pytest_generate_tests(metafunc):
-    if 'rhel_contenthost' in metafunc.fixturenames:
+    content_host_fixture = ''.join(
+        [i for i in ['rhel_contenthost', 'content_hosts'] if i in metafunc.fixturenames]
+    )
+    if content_host_fixture in metafunc.fixturenames:
```

Comment:
@JacobCallahan I'd like your thoughts here, because I think this is an anti-pattern, trying to statically set a list of function names here that should have parametrization applied during test generation.

I'm not sure markers will really work for this, because if we apply the mark within the fixture function it won't be added to the node until after this generation function is complete.

Reply:
@mshriver certainly a valid concern.
@akhil-jha is there another way you could do this?